docs/user-guide/features/multi-tenancy-setup.md

```diff
@@ -0,0 +1,75 @@
+# Multi-Tenancy Setup Guide
+
+This doc describes how to setup multi-tenancy on a testing cluster.
+
+## Tenant Bootstrap
+Tenant can be created using the following command:
+
+```bash
+kubectl create tenant $[tenant name]
+```
+
+When a new tenant is created, a cluster role and rolebinding is automatically created for this tenant for user "admin". This admin cluster role allows full access to this tenant's space resource management.
+
+In order to access the cluster, a certificate can be used to authenticate and authorize as user admin for this tenant.   
+
+## Set Up Tenant For Testing
+
+[Two scripts](../../../hack/setup-multi-tenancy) are provided to quickly set up tenant on an Arktos cluster. There are two steps involved:
+
+### 1. Create A Tenant
+
+```bash
+./create_tenant.sh mycompany
+setting storage cluster to system
+tenant/futurewei created
```

Comment:
the output should be "tenant/mycompany created".

Reply:
fixed here and other 2 places